changelog/unreleased/pull-3612

```diff
@@ -0,0 +1,7 @@
+Enhancement: Add GCS option for encryption keys
+
+These changes allow users to provide 256-bit AES crypto 

```

Comment:
Please word it something like `We have added support for xyz`. "These changes" is not meaningful when reading the changelog.

Reply:
Changed.